common/app/common/models/TitleFormFieldBean.java

```diff
@@ -0,0 +1,48 @@
+package common.models;
+
+import common.contexts.UserContext;
+import common.template.i18n.I18nIdentifier;
+import common.template.i18n.I18nResolver;
+import io.sphere.sdk.models.Base;
+import play.Configuration;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+import static java.util.Collections.emptyList;
+import static java.util.stream.Collectors.toList;
+
+public class TitleFormFieldBean extends Base {
+
+    private static final String CONFIG_TITLE_OPTIONS = "form.titles";
+
+    private List<SelectableBean> list;
+
+    public TitleFormFieldBean() {
+    }
+
+    public TitleFormFieldBean(@Nullable final String selectedTitle, final UserContext userContext,
+                              final I18nResolver i18nResolver, final Configuration configuration) {
```

Comment:
Can we have a constructor that does not depend on Configuration?

Configuration is cool, but hard to know what you exactly use from it.

I like this class https://github.com/sphereio/jvm-trainings/blob/master/tarent-web/src/main/java/io/sphere/demo/tarent/commercetools/CommercetoolsPlatformConfig.java since it is very declarative extracting values from properties. Maybe there is something doable with guice and Configuration?